src/core/connections.jl

```diff
@@ -262,6 +249,26 @@ function set_external_param!(md::ModelDef, name::Symbol, value::ModelParameter)
     md.external_params[name] = value
 end
 
+function set_external_param!(md::ModelDef, name::Symbol, value::Number; param_dims::Array{Symbol} = nothing)
```

Comment:
For `param_dims::Array{Symbol} = nothing` to work with `nothing` as a default value, you'll need to define the type as `param_dims::Union{Void, Array{Symbol}} = nothing`